src/AWS/Config.elm

```diff
@@ -8,11 +8,27 @@ type alias Credentials =
     }
 
 
+type alias EndpointPrefix =
+    String
+
+
+type alias Region =
+    String
+
+
+type alias Host =
+    String
+
+
+type Endpoint
+    = RegionalEndpoint EndpointPrefix Region
+    | GlobalEndpoint Host
```

Comment:
Region is wrapped up in `RegionalEndpoint`, and `GlobalEndpoint` doesn't need it